tests/post_training/model_scope.json

```diff
@@ -184,6 +184,9 @@
         }
     },
     "crossvit_9_240": {
+        "skipped_by_backends": {
+            "TORCH": "Wrong quantization config, model is validated by TORCH_PTQ backend"
```

Comment:
How linked config for TORCH and validation on TORCH_PTQ?

And what about another transformer models with same error, like levit_128?
```
RuntimeError: No match has been found among the model operations for the following ignored/target scope definitions:
- ignored_scope: ['{re}.*Embeddings.*', '{re}.*__truediv__*', '{re}.*matmul_1']
 Refer to the original_graph.dot to discover the operations in the model currently visible to NNCF and specify the ignored/target scopes in terms of the names there.
```

And is it planning to fix?

Reply:
Comments are addressed, please check